locations/storefinders/locally.py

```diff
@@ -31,6 +31,7 @@ def parse(self, response):
             self.post_process_item(item, store)
             yield item
 
+    # TODO: Rename to pre_process_data and arguments pattern?
```

Comment:
Yes, I think we should do this at some point. It doesn't necessarily need to be in this PR though.

Reply:
https://github.com/alltheplaces/alltheplaces/pull/9451